IFC4x3/Properties/q/QuantityOfEnergyAbsorbed_2G1OIQuh92PA5yBaw72mtx/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="QuantityOfEnergyAbsorbed" UniqueId="9005849a-e2b2-4264-a17c-2e4e870b0dfb" PrimaryDataType="IfcEnergyMeasure" />
```

Comment:
Is this somehow different to `EnergyAbsorption`?

Reply:
It is removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/168/commits/791b52ae8b05f2e9592ef4fdee6e010917dd44e1